util.py

```diff
@@ -0,0 +1,12 @@
+def convert_str_to_bool(value: str, default: bool) -> bool:
+    try:
+        if value is None:
+            return default
+        if value.lower() == 'true':
+            return True
+        elif value.lower() == 'false':
+            return False
+        else:
+            return default
+    except Exception as e:
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/rkj6124/test/pull/1#issuecomment-2370976797>#e5e09c</a><span> - 09/24/2024, 11:23 am</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> Do not catch blind exception: 'Exception'
</div>
<div id="fix">
<b>Fix:</b> Catch specific exceptions instead of a broad Exception
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -9,5 +9,5 @@
      if value.lower() == 'false':
          return False
      return default
 -except Exception as e:
 +except ValueError:
      return default
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged